.travis.yml

```diff
@@ -6,13 +6,21 @@ addons:
       - bison
       - git
       - gperf
+      - redis-server
```

Comment:
I've added this line to update redis-server package. By default, Travis CI provides redis-server v4.0.6. It doesn't support `redis-cli --cluster create` command.